extension/content.js

```diff
@@ -113,6 +113,24 @@ function addTrendingMenuItem() {
 	`);
 }
 
+function addYoursMenuItem() {
+	const [, page] = location.pathname.split('/');
+	const username = getUsername();
+	const $menu = $('.subnav-links');
+
+	if ($(`.subnav-item[href="/${page}?q=is%3Aopen+is%3Aissue+user%3A${username}"]`).length > 0) {
+		return;
+	}
+
+	const yoursMenuItem = $(`<a href="/${page}?q=is%3Aopen+is%3Aissue+user%3A${username}" class="subnav-item">Yours</a>`);
```

Comment:
Updated to use the `pageDetect` functions to set it. Passing a string in four places didn't feel DRY.